### ISSUE TITLE
Question: how to use darts.metrics.metrics.mase with gridsearch?

### ISSUE THREAD

**Issue by zora-no** [unit8co/darts#1095]
For gridsearch, the parameter "metric" requires a "function that takes two TimeSeries instances as inputs (actual and prediction, in this order), and returns a float error value". Is there any way to run a gridsearch with MASE, since it requires in-sample predictions as input?

**Comment by OmarHory** [comment]
Same issue.

**Comment by hrzn** [comment]
There's currently no out-of-the box way to use MASE with gridsearch.
However if you have any serious need for hyper-parameter search, I'd recommend you either implement your own gridsearch (it's just a for loop, really), or (better) use some hyper-parameter optimization library; see an example [here](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html#hyperparameter-tuning-with-ray-tune)
